src/helpers/getDefaultS3DataFromNamespace.ts

```diff
@@ -25,16 +27,18 @@ const getDefaultS3DataFromNamespace = async (
    * otherwise the current owner is an organization so we use OrganizationApi
    * to get the org's data
    */
-  const getOwnerData = (): Promise<any> =>
+  const getOwnerData = (): Promise<AxiosResponse<User | Organization>> =>
     isOwnerOrganization
       ? orgTileDBAPI.getOrganization(owner)
       : userTileDBAPI.getUser();
   const ownerResponse = await getOwnerData();
 
   return {
-    default_s3_path: ownerResponse.data.default_s3_path,
-    default_s3_path_credentials_name:
-      ownerResponse.data.default_s3_path_credentials_name
+    default_s3_path:
+      (ownerResponse.data.asset_locations?.notebooks as any) ||
```

Comment:
This shouldn't work. `asset_locations` contains [StorageLocation](https://github.com/TileDB-Inc/TileDB-Cloud-API-Spec/blob/master/openapi-v1.yaml#L1511) objects.
This should be something like:
```javascript
default_s3_path: ownerResponse.data.asset_locations?.notebooks?.path || ownerResponse.data.default_s3_path,
default_s3_path_credentials_name: ownerResponse.data.asset_locations?.notebooks?.credentials_name || ownerResponse.data.default_s3_path_credentials_name,
```

Reply:
Thanks for the clarification